code_gen_tester/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: code_gen_tester
 version: 0.0.1
 environment:
-    sdk: '>=2.0.0-dev.68.0 <3.0.0'
+    sdk: '>=2.1.0 <3.0.0'
 dependencies:
     source_gen: ^0.9.4+1
     build: '>=1.1.2 <2.0.0'
-    analyzer: '>=0.35.4 <0.36.0'
+    analyzer: '>=0.35.4 <1.0.0'
```

Comment:
```suggestion
    analyzer: '>=0.39.0 <1.0.0'
```

Reply:
Do you really want to go that high immediately? My original intention is to have range quite big to be a little bit "backward compatible", but I'm fine going godspeed. And if so, there's another deprecated field (`displayName`) that I will upgrade as well.

Reply:
I don't really have the time to maintain a wide range of versions, so yes